### ISSUE TITLE
how to get pretrain model? 

### ISSUE THREAD

**Issue by ailihong** [BeloborodovDS/MobilenetSSDFace#3]
hello,BeloborodovDS
    Thank you very much for your excellent work. I want to konw how to get pretrain model(face_init_full.caffemodel), train on imagenet or from mobilenetv1? 
   thanks again.

**Comment by BeloborodovDS** [comment]
@ailihong 
All right, so the model originates from here: https://github.com/chuanqi305/MobileNet-SSD, it was pretrained on VOC. In my repo it is located in models/ssd_voc/mobilenet_iter_73000.caffemodel. Note that it is configured for VOC detection (several classes, large objects).

Then it is transformed to models/ssd_face/face_init_full.caffemodel. Some differences include: just one class, smaller boxes, weights from last layers are lost. 

Then it is fine-tuned, and the result of it is models/ssd_face/best_bn_full.caffemodel, which is then deployed by merging BatchNorm with convolutions.

**Comment by ailihong** [comment]
@BeloborodovDS 
  I got it.Thank again for your excellent work.
